ansible/group_vars/wakuv2-staging.yml

```diff
@@ -0,0 +1,105 @@
+  - { name: ivan,    uid: 8006, groups: ['docker', 'dockremap'], key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJBdm8y1PfWjT1pioaWJSZ2ETrUySb+dS/ifDg+VIpLY ivansete@status.im' }
+
+# Tag dependent on fleet: test, prod, staging
+nim_waku_cont_tag: 'deploy-wakuv2-{{ stage }}'
+nim_waku_cont_name: 'nim-waku-v2'
+nim_waku_log_level: 'debug'
+nim_waku_protocols_enabled: ['archive', 'relay', 'store', 'filter', 'lightpush', 'rln-relay', 'peer-exchange']
+nim_waku_disc_v5_enabled: true
+nim_waku_dns4_domain_name: '{{ dns_entry }}'
+nim_waku_node_key: '{{lookup("bitwarden", "fleets/wakuv2/"+stage+"/nodekeys", field=hostname)}}'
+
+# Topic configuration
+nim_waku_topics:
+  - '/waku/2/rs/18/16'
+  - '/waku/2/rs/18/17'
+  - '/waku/2/rs/18/18'
+  - '/waku/2/rs/18/128'
+  - '/waku/2/rs/18/129'
+  - '/waku/2/rs/18/130'
+  - '/waku/2/rs/18/768'
+  - '/waku/2/rs/18/769'
+  - '/waku/2/rs/18/770'
+  - '/waku/2/rs/18/896'
+  - '/waku/2/rs/18/897'
+  - '/waku/2/rs/18/896'
+
+
+# Ports
+nim_waku_p2p_tcp_port: 30303
+nim_waku_p2p_udp_port: 30303
+nim_waku_metrics_port: 8008
+nim_waku_disc_v5_port: 9000
+nim_waku_websock_port: 8000
+nim_waku_rpc_tcp_port: 8545
+
+# Limits
+nim_waku_p2p_max_connections: 300
+
+# Store
+nim_waku_store_message_retention_policy: 'time:2592000' # 30 days
+
+# DNS Discovery
+nim_waku_dns_disc_enabled: true
+nim_waku_dns_disc_url: 'enrtree://AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM@test.waku.nodes.status.im'
+
+# Enable WebSockets via Websockify
+nim_waku_websockify_enabled: false
+
+# Enable websockets in Waku
+nim_waku_websocket_enabled: true
+nim_waku_websocket_secure_enabled: true
+nim_waku_websocket_domain: '{{ dns_entry }}'
+nim_waku_websocket_ssl_dir: '/etc/letsencrypt'
+nim_waku_websocket_ssl_cert: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain }}/fullchain.pem'
+nim_waku_websocket_ssl_key: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain }}/privkey.pem'
+
+# Waku rln-relay parameters
+nim_waku_rln_relay_content_topic: '/toy-chat/3/mingde/proto'
+nim_waku_rln_relay_dynamic: true
+nim_waku_rln_relay_eth_contract_address: '0x9C09146844C1326c2dBC41c451766C7138F88155'
+nim_waku_rln_relay_eth_client_address: 'ws://linux-01.ih-eu-mda1.nimbus.sepolia.wg:9557'
+
+# Consul Service
+nim_waku_consul_success_before_passing: 5
+nim_waku_consul_failures_before_warning: 2
+nim_waku_consul_failures_before_critical: 20
+
+# Peer connecting
+waku_peers_rpc_port: '{{ nim_waku_rpc_tcp_port }}'
+waku_peers_rpc_timeout: 20
+waku_peers_rpc_retries: 5
+waku_peers_consul_services:
+  - { name: '{{ nim_waku_cont_name }}', env: '{{ env }}',  stage: '{{ stage }}' }
+
+# LetsEncrypt via Certbot
+certbot_docker_enabled: true
+certbot_admin_email: 'devops@status.im'
+certbot_containers_to_stop: ['{{ nim_waku_cont_name }}']
+certbot_certs:
+  - domains: [ '{{ nim_waku_websockify_domain }}' ]
+
+# Connection limits
+conn_limit_comment: 'WebSocket conn limit'
```

Comment:
Do we want the connection limit applied to this fleet as well?

Reply:
I'm not sure... What's the rationale for not having a limit?

Reply:
Ask @jm-clius, he said we don't need it.

Reply:
We now have internal connection limits applied in peer management. While this feature is still being hardened, it's worth keeping the host limits on the fleets used by external websocket clients as a sanity safety check, but it's unnecessary for the new staging fleet that will not be subscribed to the default pubsub topic(s).